### PR TITLE
add "aspect-ratio-16x9" or "rectangle" icon

### PR DESCRIPTION
**Pull request by Reund0**
Hi there.
I was creating buttons for my image editor and needed a rectangle to show 16:9 cropping option.
But when I searched the Lucide icons, there wasn't any rectangle or aspect-ratio icon.
So I open a PR for this icon. 

PS: aspect ratio of the rectangle icon is 16:9.

### CODE REVIEW

File: icons/aspect-ratio-16x9.svg [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <rect x="2" y="6.44" width="20" height="11.25" rx="2"/>
```
Comment (karsa-mistmere) [inline icons/aspect-ratio-16x9.svg:12]: This looks really ugly on low dpi, while it's not a true 16:9 aspect ratio, I suggest to go with the one below, and just call it `rectangle-horizontal`.
```suggestion
  <rect x="2" y="6" width="20" height="12" rx="2"/>
```

image

Comment (Reund0) [reply icons/aspect-ratio-16x9.svg]: Yes, you're right. I change the pixel sizes and also add a vertical version of the icon but it seems there are problems.

File: tags.json [hunk 2]
```diff
@@ -2950,6 +2950,18 @@
     "music",
     "connect"
   ],
+  "rectangle-horizontal": [
+    "rectangle",
+    "aspect ratio",
+    "16:9",
+    "horizontal",
```
Comment (ericfennis) [inline tags.json:2957]:
```suggestion
    "horizontal"
```

**Review by ericfennis — CHANGES_REQUESTED** [review]
Remove trailing commas in JSON file

File: icons/aspect-ratio-16x9.svg (hunk 1 above)
Comment (karsa-mistmere) [reply icons/aspect-ratio-16x9.svg]: Upon further consideration, while I think such rectangle icons would be useful, especially as building blocks of future icons, I'd prefer to keep them free of any explicit aspect ratio, such as: 
image

**Comment by Reund0** [comment]
Oh sorry i forgot. It's done.

File: icons/rectangle-vertical [hunk 3]
```diff
@@ -0,0 +1,13 @@
+<svg
```
Comment (ericfennis) [inline icons/rectangle-vertical:1]: This file is missing the `.svg` extension

**Review by ericfennis — CHANGES_REQUESTED** [review]

**Review by ericfennis — APPROVED** [review]

**Pull request merged**